python/ray/tests/test_multi_node.py

```diff
@@ -201,6 +202,70 @@ def f():
         assert out.count(log_message) == 4
 
 
+def test_logging_to_multiple_drivers(call_ray_start):
+    redis_address = call_ray_start
+
+    driver_script = """
+from __future__ import print_function
+import ray
+from ray.tests.utils import CaptureOutputAndError
+import sys
+import time
+import atexit
+
+atexit.register(lambda: print("error"))
+
+ray.init(redis_address="{}", log_to_driver=True)
+
+log_range = {}
+total_range = {}
+
+@ray.remote
+def f():
+    # It's important to make sure that these print statements occur even
+    # without calling sys.stdout.flush() and sys.stderr.flush().
+    for i in range(*log_range):
+        print(i)
+        time.sleep(0.01)
+
+captured = dict()
+with CaptureOutputAndError(captured):
+    ray.get(f.remote())
+    time.sleep(1)
+
+output_lines = captured["out"]
+output = [line.split(" ")[1] for line in output_lines.splitlines()]
+
+for i in range(*total_range):
+    if i >= log_range[0] and i < log_range[1]:
+        if str(i) != output[i - log_range[0]]:
+            sys.exit()
+    else:
+        if str(i) in output:
+            sys.exit()
+
+error_lines = captured["err"]
+if len(error_lines) != 0:
+    sys.exit()
+
+print("success")
```

Comment:
We aren't checking for this anywhere are we? If not, then remove it.

Reply:
This is actually checked for in `wait_for_success_output`. I think `test_drivers_release_resources` uses a similar check to determine if the test passed.